packages/carousel/package.json

```diff
@@ -71,13 +71,13 @@
     "react-dom": "^17.0.2",
     "require-from-string": "2.0.2",
     "rimraf": "^3.0.2",
-    "styled-components": "^5.3.5",
+    "styled-components": "^5.3.9",
     "typescript": "^4.5.4"
   },
   "peerDependencies": {
-    "react": ">= 16.10.0",
-    "react-dom": ">= 16.10.0",
-    "styled-components": ">=5 <5.3.4",
-    "pcln-design-system": "5.6.1"
+    "react": ">=16.10.0",
+    "react-dom": ">=16.10.0",
+    "styled-components": ">=5 <5.3.4 || >=5.3.7 <6",
```

Comment:
is this specific to this package or should it be added to all?

Reply:
I added this because 5.3.7 removed the Ukraine messaging that was tagged as a supply chain attack.